src/sagemaker/chainer/estimator.py

```diff
@@ -162,6 +162,7 @@ def create_model(
         entry_point=None,
         source_dir=None,
         dependencies=None,
+        **kwargs
```

Comment:
is there something here that should actually get used?

Reply:
I am only fixing TensorFlow and SKLearn for now. I think all other frameworks needs more fixes than this one(similar to the fixes I added to sklearn). That work should be properly scheduled and we should add integ tests as well.